### ISSUE TITLE
Cannot create bindings

### ISSUE THREAD

**Issue by TomGrill** [mk-5/gdx-fireapp#1]
Hi I did as you suggest in the wiki: https://github.com/mk-5/gdx-fireapp/wiki/iOS-Firebase-SDK-installation step4

When i try to create the bindings I get this error:

```
fatal error: 'FirebaseAnalytics/FirebaseAnalytics.h' file not found
```

More detailed:

```
2017-10-04 12:51:55:787 +0200 [main] DEBUG org.moe.natjgen.Main - NatJGen started
2017-10-04 12:51:56:466 +0200 [main] DEBUG org.moe.natjgen.Indexer - Clang options: [-arch, arm64, -x, objective-c, -fmessage-length=133, -std=c99, -fobjc-arc, -fpascal-strings, -DNS_BLOCK_ASSERTIONS=1, -DOBJC_OLD_DISPATCH_PROTOTYPES=0, -isysroot, /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk, -miphoneos-version-min=9.0, -I/var/folders/m0/mc2v3k1d16b5ftc1n59dmgz40000gn/T/NatJGen354660108793050899, -F/Users/tom/fireapp/ios-moe/xcode/Pods/FirebaseCore/Frameworks, -F/Users/tom/fireapp/ios-moe/xcode/Pods/FirebaseAnalytics/Frameworks/frameworks, -F/Users/tom/fireapp/ios-moe/xcode/Pods/FirebaseAuth/Frameworks/frameworks, -F/Users/tom/fireapp/ios-moe/xcode/Pods/FirebaseDatabase/Frameworks, -F/Users/tom/fireapp/ios-moe/xcode/Pods/FirebaseStorage/Frameworks/frameworks, -F/Users/tom/fireapp/ios-moe/xcode/Pods/FirebaseMessaging/Frameworks/frameworks, -F/Users/tom/fireapp/ios-moe/xcode/Pods/FirebaseCrash/Frameworks]
/var/folders/m0/mc2v3k1d16b5ftc1n59dmgz40000gn/T/tempfile7002671322241956770.m:3:9: fatal error: 'FirebaseAnalytics/FirebaseAnalytics.h' file not found
2017-10-04 12:51:57:347 +0200 [main] DEBUG org.moe.natjgen.Indexer - Indexing translation unit
2017-10-04 12:52:01:357 +0200 [main] DEBUG org.moe.natjgen.Indexer - Clang options: [-arch, armv7, -x, objective-c, -fmessage-length=133, -std=c99, -fobjc-arc, -fpascal-strings, -DNS_BLOCK_ASSERTIONS=1, -DOBJC_OLD_DISPATCH_PROTOTYPES=0, -isysroot, /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk, -miphoneos-version-min=9.0, -I/var/folders/m0/mc2v3k1d16b5ftc1n59dmgz40000gn/T/NatJGen354660108793050899, -F/Users/tom/fireapp/ios-moe/xcode/Pods/FirebaseCore/Frameworks, -F/Users/tom/fireapp/ios-moe/xcode/Pods/FirebaseAnalytics/Frameworks/frameworks, -F/Users/tom/fireapp/ios-moe/xcode/Pods/FirebaseAuth/Frameworks/frameworks, -F/Users/tom/fireapp/ios-moe/xcode/Pods/FirebaseDatabase/Frameworks, -F/Users/tom/fireapp/ios-moe/xcode/Pods/FirebaseStorage/Frameworks/frameworks, -F/Users/tom/fireapp/ios-moe/xcode/Pods/FirebaseMessaging/Frameworks/frameworks, -F/Users/tom/fireapp/ios-moe/xcode/Pods/FirebaseCrash/Frameworks]
/var/folders/m0/mc2v3k1d16b5ftc1n59dmgz40000gn/T/tempfile7002671322241956770.m:3:9: fatal error: 'FirebaseAnalytics/FirebaseAnalytics.h' file not found

```

This is the app I am working with to test: Simple gds app.
https://github.com/TomGrill/gdx-fireapp-test

**Comment by mk-5** [comment]
Hi! Thanks for reporting! 
and nice to meet you! I use some of your libs in my projects.

It is a problem with framework path inside firebase.nbc file. 
Your Pods is installed inside
`FirebaseAnalytics/Frameworks/FirebaseAnalytics.framework` - 

and path inside **firebase.nbc** is: 
`FirebaseAnalytics/Frameworks/frameworks/FirebaseAnalytics.framework`.
Is same issue for FirebaseStorage, FirebaseAuth, FirebaseMessaging

Please try to correct this paths - and then bindings will be generated. 
I send pull-request to gdx-fireapp-test repo with it.

I don't know exactly why it was happen, and why this paths are different sometimes.
I will write a note about it at Wiki page until i figure out whats going on and i will  do some extra tests. If correct paths are without `frameworks` directory inside - i will correct `firebase.nbc` paths in repo.

Which version of Pods do you have? (just call `pod --version` in terminal)

**Comment by TomGrill** [comment]
pod --version: 1.3.1

Currently trying the PR

**Comment by TomGrill** [comment]
Ok, this are my resulsts.

Manually adjusting the correct paths did work. Not sure why the wrong paths appeared.

After I pulled your PR it still did not work. I adjusted the paths manually in the nbc windows and then everything went through. Kinda strange.

Maybe some invisible specialchars cause the problem because the manually working file is exactly the same you served with the PR.

**Comment by TomGrill** [comment]
Ok seems to work so far. I like the using the pod command, I will try to implement that in my extensions as well.

**Comment by mk-5** [comment]
Mhmm yes, this is strange. 

I installed pods via pod version 1.1.1.
I will do some tests with it, but at this point notes at wiki page must be enough.

**Comment by mk-5** [comment]
The pods frameworks paths was corrected in the 1.2.0 release, so i close this issue.
